core/domain/interaction_jobs_one_off.py

```diff
@@ -152,6 +152,39 @@ def reduce(key, values):
         yield (key, values)
 
 
+class MultipleItemInteractionLtOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """Audit job that flags MultipleItemInteractions with too many choices.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        exploration = exp_fetchers.get_exploration_from_model(item)
+        interactions_to_check = ('MultipleChoiceInput', 'ItemSelectionInput')
+        for state in exploration.states.items():
+            if state[1].interaction.id in interactions_to_check:
```

Comment:
Why `state[1]`?

Reply:
`state `is a tuple represented as` (state_name, state_properties)` and thus state_properties being at index 1, I have used `state[1].`